src/agent/onefuzz-supervisor/src/worker.rs

```diff
@@ -239,6 +239,33 @@ impl IWorkerRunner for WorkerRunner {
     }
 }
 
+trait SuspendableChild {
+    fn suspend(&mut self) -> Result<()>;
+}
+
+#[cfg(target_os = "windows")]
+impl SuspendableChild for Child {
+    fn suspend(&mut self) -> Result<()> {
+        let result = unsafe { winapi::um::debugapi::DebugActiveProcess(self.id() as u32) };
```

Comment:
Interesting. For others, can you please document why we're using this as a workaround?